pyproject.toml

```diff
@@ -14,17 +14,14 @@ pydis_core = { version = "9.9.2", extras = ["async-rediscache"] }
 aiohttp = "3.8.4"
 arrow = "1.2.3"
 beautifulsoup4 = "4.12.2"
-colorama = { version = "0.4.6", markers = "sys_platform == 'win32'" }
 coloredlogs = "15.0.1"
 deepdiff = "6.3.0"
 emoji = "2.6.0"
 feedparser = "6.0.10"
-lxml = "4.9.3"
```

Comment:
We use lxml as the parser for parsing xml files using `BeautifulSoup` in two places https://github.com/search?q=repo%3Apython-discord%2Fbot%20lxml&type=code

Reply:
Ok, I  saw that one. I just didn't think that it would be an extension that required a package.

Reply:
Yea, it requires you to choose the parser and install separately. https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser

Reply:
Fixed in the forcepush. 9743c5b